day1/type conversion/ans3.js

```diff
@@ -0,0 +1,13 @@
+/*3. Declare a variable let arraySize = 25;. Using a for loop, add numbers from one onwards into 
+an array till the arraySize is reached.  */
+
+
+let arraySize=25
+
+let arr=[]
+
+for(let i=0;i<arraySize;i++){
+    arr.push(i)
```

Comment:
The question says to add numbers from one onwards, and as per your logic, you are adding 0 in the array.